src/test/java/de/tum/cit/aet/artemis/util/RequestUtilService.java

```diff
@@ -782,6 +782,31 @@ public <T> void delete(String path, HttpStatus expectedStatus, T body) throws Ex
         restoreSecurityContext();
     }
 
+    /**
+     * Sends a multipart put request with a mandatory json file and an optional file.
```

Comment:
not a PUT request, right?

Reply:
thanks for noticing that! I updated the comment